i18n/es/translation.json

```
@@ -466,5 +459,27 @@
   "Theme": "Tema",
   "Language": "Idioma",
   "Settings": "Configuración",
-  "The Health Factor represents how “safe” your leverage portfolio is, defined as the risk-adjusted proportion of collateral deposited versus the borrowed risk-adjusted amount. A health factor below 1x will be considered with a shortfall and open to liquidation.": "El Factor de Salud representa cuán “seguro“ es tu portafolio con apalancamiento, definido como la proporción ajustada al riesgo del colateral depositado frente al monto ajustado al riesgo que se ha tomado prestado. Un factor de salud por debajo de 1x se considerará con un déficit y estará abierto a la liquidación."
+  "The Health Factor represents how “safe” your leverage portfolio is, defined as the risk-adjusted proportion of collateral deposited versus the borrowed risk-adjusted amount. A health factor below 1x will be considered with a shortfall and open to liquidation.": "El Factor de Salud representa cuán “seguro“ es tu portafolio con apalancamiento, definido como la proporción ajustada al riesgo del colateral depositado frente al monto ajustado al riesgo que se ha tomado prestado. Un factor de salud por debajo de 1x se considerará con un déficit y estará abierto a la liquidación.",
+  "Token allowance surplus after operation": "Excedente de permisos de token luego de la operación",
+  "up to": "hasta",
+  "Refinance Loans": "Refinancia tus préstamos",
+  "FROM": "Desde",
+  "Deposit EXA on Extra Finance": "Deposita EXA en Extra Finance",
+  "Deposit EXA on Extra Finance and earn interest on it.": "Deposita EXA en Extra Finance y gana intereses sobre él.",
+  "Go to Extra Finance": "Ir a Extra Finance",
+  "Reduce Exposure": "Reduce tu exposición",
+  "Provide Liquidity on Velodrome": "Proporciona Liquidez en Velodrome",
+  "Provide liquidity to the EXA/wETH pool.": "Proporciona liquidez al pool EXA/wETH.",
+  "Go to Velodrome": "Ir a Velodrome",
+  "Bridge & Swap with Socket": "Bridge & Swap con Socket",
+  "Seamlessly bridge and swap assets to OP Mainnet from many different networks.": "Pase y cambie activos de manera fluida a OP Mainnet desde muchas redes diferentes.",
+  "Cross Network": "Red Cruzada",
```

I'm not sure about this, but the translation sounds weird
```suggestion
  "Cross Network": "Entre Redes",
```